docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md

```diff
@@ -30,7 +30,11 @@ The above statement creates an index for the _degree_ property on all edges carr
 
 ### Create Composite Index
 
-The schema indexes also support spawning over multiple properties. An index on multiple properties for all vertices that have a particular tag is called a composite index. Consider the following example:
```

Comment:
Schema indexes support indexing multiple attributes simultaneously. The index that includes multiple attributes is called a composite index in **Nebula Graph**.